variables.tf

```diff
@@ -145,6 +145,12 @@ variable "kubernetesVersion" {
   default     = "1.23.12"
 }
 
+variable "keyVaultPurgeProtection" {
+  type        = bool
+  description = "Specifies whether the Key vault purge protection is enabled."
+  default     = false
```

Comment:
In my opinion, we should set the default value to `true`, as this is the securer setting.